lib/test_shibboleth_headers.rb

```diff
@@ -13,7 +13,7 @@ def initialize(app)
 
   def call(env)
     env['REMOTE_USER'] = user
-    # env['eduPersonEntitlement'] = Array(groups).join(';')
+    env['eduPersonEntitlement'] = groups.join(';') if groups.present? # allow omission of headeer for testing
```

Comment:
Typo: `s/headeer/header/`

Reply:
good catch, thanks! (my old laptop keyboard is flaking, and giving me enough double-presses that i'm pretty sure it's not just my own clumsiness)

Reply:
ha!